cypress/locators/alert/locators.js

```diff
@@ -0,0 +1,4 @@
+// component preview locators
+export const ALERT_CHILDREN = 'div[height][style="height: calc(100% - 76px);"]';
```

Comment:
why `style="height: calc(100% - 76px)` locator?

Reply:
yes, because we have no other attributes to locate children text in Alert